package.json

```diff
@@ -10,7 +10,7 @@
     "@babel/core": "^7.12.3",
     "@babel/preset-env": "^7.12.1",
     "@babel/preset-typescript": "^7.12.1",
-    "@tsconfig/node12": "^1.0.7",
+    "@tsconfig/node16": "^1.0.3",
```

Comment:
I'm 99% sure that this package isn't even required. It's not imported into the TS config, and it's not referenced _anywhere_. Want to just remove it? :)

Reply:
ohh, cool. I will remove it

Reply:
@bkendall now removed `@tsconfig/node16` and I squashed my commits. Ready for another review if you are free.